packages/render-fidelity-tools/src/common.ts

```diff
@@ -95,6 +95,7 @@ export interface ScenarioConfig {
   verticalFoV: number;
   exclude?: Array<string>;
   renderSkybox: boolean;
+  realTimeFiltering: boolean;
```

Comment:
Considering that this configuration is specific to babylon, I don't think it should live in the common config file. You'll notice I have a special case in our Filament renderer for the directional light test. I would do something similar here and set this based on name-matching the IBL file.

Reply:
Yeah that's much better. I just saw dspbr-pt put some config variables in the scenario config so i thought that's ok. Anyway your way is more organized . Thanks!

Reply:
Your point on `pt` is a good one. Do we actually use different numbers of samples for different scenarios? I would prefer to make that a constant inside that renderer for the same reason if possible.